client/state/happychat/connection/actions.js

```diff
@@ -154,6 +156,8 @@ export const receiveError = ( error ) => ( { type: HAPPYCHAT_IO_RECEIVE_ERROR, e
  * Returns an action object for the transcript reception.
  *
  * @param {object} result An object with {messages, timestamp} props
+ * @param {Array} result.messages An array of message objects
+ * @param {number} result.timestamp The transcript reception timestamp
  * @returns {object} Action object
```

Comment:
Nit: I know this isn't a requirement, but IMHO it really makes things more readable when the different sections of the JSdoc params are vertically aligned to each other:

```suggestion
 * @param {object} result           An object with {messages, timestamp} props
 * @param {Array}  result.messages  An array of message objects
 * @param {number} result.timestamp The transcript reception timestamp
 * @returns {object} Action object
```

Feel free to ignore this nit of the nits 😉